bfgrubcheck

```diff
@@ -76,14 +83,16 @@ elif grep -q "password admin BlueField" $GRUBCFGFILE ; then
     rm -f $GRUBPASSWDFILE.tmp
     rm -f $GRUBPASSWDFILE
     RET=1
-    tmpfile=$(mktemp /tmp/bfgrubcheck.XXXXXX)
     until [ ${RET} -eq 0 ]; do
```

Comment:
Looks like this file has the updated version.
Still a question here, is the loop because that the "$GRUBPASSWDCMD" could fail?

Reply:
See above.  The tmpfile has been removed and the check simplified.